demo/main.ts

```diff
@@ -115,7 +116,12 @@ function jsonXmlReplacer(_key: string, value: any): any {
 async function runUpdatingXQuery(script: string) {
 	const result = await fontoxpath.evaluateUpdatingExpression(script, xmlDoc, null, null, {
 		debug: true,
-		disableCache: true
+		disableCache: true,
+		logger: {
+			trace: m => {
+				traceOutput.innerHTML = m;
```

Comment:
I would append here, not overwrite.

Reply:
Also, let's add a console.log here

Reply:
consider using textContent instead of innerHTML to avoid having to escape things that look like HTML